kernelci/data/kernelci_api.py

```diff
@@ -94,6 +103,45 @@ def get_root_node(self, node_id):
         resp = self._get('/'.join(['get_root_node', node_id]))
         return json.loads(resp.text)
 
+    def pubsub_event_filter(self, sub_id, event):
+        """Filter Pub/Sub events
+
+        Filter received Pub/Sub event using provided filter dictionary.
+        Return True if the event matches with the filter, otherwise False.
+        """
+        event_filter_status = True
```

Comment:
This variable isn't actually used, because the method just returns every time it finds a filter that doesn't match the data. So I think we can drop it.
```suggestion
```